code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -26,6 +26,23 @@
 	playsound(src.loc, "rustle", 50, 1, -5)
 	return ..()
 
+/obj/item/storage/backpack/examine(mob/user)
+	var/space_used = 0
+	for(var/obj/item/I in contents)
+		space_used += I.w_class
+	if(!..(user, 1))

```

Comment:
I'd maybe move this above the `for` loop, cause if they're too far away you don't need the for loop to happen at all